frontend/packages/common-frontend/package.json

```
@@ -45,7 +45,7 @@
     "access": "public"
   },
   "dependencies": {
-    "lit": "^2.0.0-rc.2"
+    "lit": "2.0.0"
```

I guess it should have `^`.
```suggestion
    "lit": "^2.0.0"
```

---

Probably, it would be even better to use a peer dependency here.

---

done